src/router/pages/Home.tsx

```diff
@@ -8,24 +8,32 @@ const KanmiList = (): React.ReactElement => {
   const kanmiList = kanmi.map((monthlyKanmi, index) => {
     const mikanList = monthlyKanmi.mikanList.map((mikan, index) => {
       return (
-        <div key={index}>
+        <Grid item xs={6} md={3} key={index}>
           <img
             className="image-mikan"
             // TODO: 清原から画像使う許可を貰うか、届いた後に自分で写真を撮り、publicディレクトリ下に配置する
             // src={`https://github.com/tofuchic/kancolle/raw/main/public/mikan/${mikan}.png`}
             src={`https://github.com/tofuchic/kancolle/raw/main/public/mikan/fruit_orange.png`}
+            style={{
+              width: '100%',
+              height: 'auto',
+            }}
           />
-        </div>
+        </Grid>
       )
     })
     return (
-      <Grid item xs={12} md={6} key={index}>
-        <h2>
-          <Link to={'/mikan?displayName=' + monthlyKanmi.mikanList}>
-            {monthlyKanmi.year}年{monthlyKanmi.month}月配送
-          </Link>
-        </h2>
-        <YokoWrapper>{mikanList}</YokoWrapper>
+      <Grid item xs={12} md={10} lg={8} xl={6} key={index}>
+        <Grid container key={index} sx={{padding: "8px"}}>
```

Comment:
```suggestion
        <Grid container key={index} sx={{ padding: '8px' }}>
```

うちのVSCodeで開くと、prettier君が勝手にこうするねん

Reply:
vscodeの保存時の設定の問題でした。保存時にprettierを自動で動作させるようにしました。